docs/application_management.md

```diff
@@ -1,27 +1,54 @@
 # Application Management
 
-Currently application creation/managment happens via the ArgoCD UI. In the future
-we will create and manage applications declaratively.
-
-## Deploying Applications with ArgoCD
-
-1. Log into ArgoCD ui at ```echo https://$(oc get routes | grep argocd-server | awk 'NR==1{print $2}')```
-2. Click create application
-3. Fill the fields out like so:
-
-```text
-Application name: <application_name>
-Project: <your_team_project>
-Sync Policy: Automatic (or manual)
-Sync Option: Check Use A Schema...
-Repository URL: <git_url>
-Revision: <branch_name> or HEAD
-Path: <path_to_overlay> or .
-Cluster: Your cluster address
-Namespace: <application_namespace>
-```
-
-Leave the rest blank and click create at the top left.
-If everything works the creation slider will go away, refresh the page and the app should be there if isn't already. Click sync and your application should be deployed.
+While ArgoCD allows you to create ArgoCD applications via the UI and CLI, it is our
+policy that all applications be created [declaratively](https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#applications).
+
+This allows us to easily retrieve your applications should the need arise. 
+
+### Create the Application
+
+In order to create an ArgoCD app first create the `Application` yaml. See below for an
+example:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: <application_name> 
+spec:
+  destination:
+    namespace: <destination_namespace>
+    server: <cluster_name> 
+  project: data-hub
+  source:
+    path: <path_to_kustomization>
+    repoURL: <repo_url>
+    targetRevision: HEAD
+  syncPolicy:
+    automated:
+      prune: true
+      selfHeal: true
+    syncOptions:
+    - Validate=false
+``` 
+
+The placeholders in this minimal spec work as follows:
+
+> `application_name` the name of the `Application` resource as well as the name as it appears on the ui.
```

Comment:
Could you just use a bulleted list instead? This may be personal preference on my part :-)

Reply:
There are really only 2 tasks - 1) create application yaml and 2) create a PR  (i.e. the headings) I could number the headings. I didn't create steps for how to create the application yaml since it's mostly just an example, it might differ for others. What do you think?

Reply:
Did you mean to comment on the `checklist` comment I had?

Reply:
(I was saying have all five of the placeholders be in a bullet list instead)

Reply:
Err yes but for the bullet list comment - I converted it to enumerated list.